### ISSUE TITLE
Схема Responce резюме из hh.ru

### ISSUE THREAD

**Issue by FordCruller** [huntflow/api#673]
Метод: https://api.huntflow.ru/v2/docs#get-/accounts/-account_id-/applicants/-applicant_id-/externals/-external_id-

Подскажите, пожалуйста схему Responce когда принимается резюме не из NATIVE источника, а из HH. Так, в этом случае например в object-data: { body: "string" } не этот код, а своя структура, похожая на json-объект resume, но во многом все же отличающаяся.

**Comment by Holm78** [comment]
Да, тоже ищу схему для источника HH, чтобы импортированное резюме было нормально размечено после импорта.
